index.js

```diff
@@ -64,3 +68,22 @@ module.exports = function setup(name, gitSha, options, process = 'http') {
 
   return beeline(config);
 };
+
+/**
+ * @type {Record<string, unknown>}
+ */
+const processInfo = {};
+
+function collectMemoryStats() {
+  const mem = process.memoryUsage();
```

Comment:
Does `process` not need to be `require`d anywhere?

Reply:
nope, it's a global